sql/updates/world/3.3.5/9999_99_99_99_world.sql

```diff
@@ -0,0 +1,7 @@
+DROP TABLE IF EXISTS `zone_default_light_override`;
+CREATE TABLE `zone_default_light_override` (
+	`zoneId` INT(10) UNSIGNED NOT NULL,
+	`defaultLight` INT(10) UNSIGNED NOT NULL,
+	`comment` VARCHAR(255) NULL DEFAULT NULL COLLATE 'utf8_general_ci',
+	PRIMARY KEY (`zoneId`)
+)COLLATE='utf8_general_ci'ENGINE=InnoDB;
```

Comment:
Single space or NewLine after the closing parenthesis ) .

Reply:
sure